### PR TITLE
ACM-7060 Refactor search definitions to reduce duplication

### PR DESCRIPTION
**Pull request by jlpadilla**
Jira Issue: https://issues.redhat.com/browse/ACM-7060

### CODE REVIEW

File: frontend/src/routes/Home/Search/searchDefinitions.tsx [hunk 1]
```diff
@@ -17,49 +17,21 @@ export const getSearchDefinitions = (t: TFunction) => {
   return {
```
Comment (jlpadilla) [inline frontend/src/routes/Home/Search/searchDefinitions.tsx:17]: ⬇️ Start review at the bottom. ⬇️

Comment (jlpadilla) [reply frontend/src/routes/Home/Search/searchDefinitions.tsx:17]: @zlayne The change set is big, but notice there's no changes to the test snapshot.  The tests for this function were great identifying problems.

Comment (zlayne) [reply frontend/src/routes/Home/Search/searchDefinitions.tsx:17]: This looks great!

**Review by zlayne — APPROVED** [review]

**Pull request merged**